.github/workflows/test-pr.yml

```diff
@@ -130,6 +130,57 @@ jobs:
           name: kore-exec.tar.gz
           path: |
             **/kore-exec.tar.gz
+      - name: 'On Success, Upload the package built to the Summary Page'
```

Comment:
Another option is set:
```retention-days: 1 ```

Reply:
This last option seems more reasonable. I'm not sure what you mean with

> which I think is not removed from the docker image it's installed 

But I don't think we should consider publishing images from PR tests as well.

Reply:
Would it be okay for you if I set the retention of the artifact for just one day? If less than this is possible, it would also be fine, as we only need it for the current execution. If not, you can say better than I do, but I don't think we have enough PR testing volume to consume all the space that we have available in one day...

Reply:
I think starting with 1 day is reasonable and see how it goes. I can elaborate further on my semi-ramble piece about not being removed from the image. 

I was a bit all over what I meant is the deb is installed to the docker image for testing with the 'test-package' action. This is usable if we published it to our internal docker image. We have the dockerhub private registry and one we use internally with our build servers. We can speak in a private forum to discuss using this.